lib/mix_gleam.ex

```diff
@@ -19,10 +19,11 @@ defmodule MixGleam do
     end)
   end
 
+  @deprecated "Please prepend :gleam to :compilers and add :\"deps.get\" to :aliases following mix_gleam README.md instead"
```

Comment:
I'm uncertain about deprecating this functionality.
I guess it depends on whether we're likely to have any other aliases in the future; I agree it isn't much use with only one alias, and it's nice to have a more explicit config in that case.

If we do deprecate this function, the https://github.com/gleam-lang/mix_gleam/blob/main/lib/mix_gleam/aliases.ex module should probably also be deprecated.

@lpil?

Reply:
My goal with this PR were:
- Make it easier for people to see what the plugin is doing and how it works. I think the `add_aliases` function obscures some of it and directly adding an alias is easier to understand.
- Bring the `mix_gleam` project a bit closer to not needing to be an archive. I think if a function from this project is being called in `mix.exs` then it needs to be an archive. Next step would be to make Mix understand gleam sub-dependencies fetching (I think?)
- Have one way to do things

That's why I think it's beneficial to deprecate this function and have users change their setup. I would consider the `MixGleam.Aliases` module to be internal and not public interface, so I don't think it needs to be deprecated

Reply:
Sounds reasonable to me.